pom.xml

```diff
@@ -957,6 +957,11 @@
                 <artifactId>jna</artifactId>
                 <version>4.5.1</version>
             </dependency>
+            <dependency>
```

Comment:
does this need to be added added to licesnses.yaml? https://github.com/apache/druid/blob/master/dev/license.md#when-you-add-a-new-library-dependency-into-druid

Reply:
It looks like this dependency is already added in licenses.yaml before according to https://github.com/apache/druid/blob/6040c30fcd29f74b9a2efb5d8cabce46972ea633/licenses.yaml#L596